manifests/site.pp

```diff
@@ -47,7 +47,7 @@
   environment => 'PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin'
 }
 
-Firewall {
+Firewall_multi {
```

Comment:
Leaving this as `Firewall` doesn't work because the `firewall_multi` resources in `post.pp` then get tangled in dependency cycles. The way I've chosen to fix this is to apply the resource default to `firewall_multi` instead; a side effect is that we mustn't use `firewall` resources directly or else they won't be subject to these ordering requirements. Should we enforce that with a pre-commit hook? Or should we try to lobby the maintainer of `firewall_multi` to forward dependencies from `firewall_multi` resources to `firewall` resources? Or should we go to the inconvenience of not using `firewall-multi` in `post.pp` (and `pre.pp` when input rules are merged in)?

Reply:
I think using `Firewall_multi` here for now is best, and then maybe try to talk to the maintainer of the package and see what they think is best? It's a concern that I think they should hopefully be receptive to.